test/controllers/file_submissions_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class FileSubmissionsControllerTest < ActionController::TestCase
+
+  def setup
+    create(:active_game)
+  end
+
+  test 'guest and user cannot access file submissions' do
+    user = create(:user)
+    challenge = create(:file_submission_challenge)
+    file_submission = create(:file_submission,
+                              challenge_id: challenge.id,
+                              user_id: user.id,
+                              submitted_bundle: File.open(Rails.root.join('test/files/test_submission.zip')),
+                              description: "Text",
+                              demoed: false
+                            )
```

Comment:
If your default are setup correctly in the Factory as described below, this should only need to be

```ruby
file_submission = create(:file_submission)
```